cvmfs/util/algorithm.h

```diff
@@ -108,6 +115,59 @@ class StopWatch : SingleCopy {
 };
 
 
+class Log2Hist
+{
+ private:
+  uint number_of_bins;
+  atomic_int32 *bins;
+  uint *boundary_values;
+  uint count_digits(uint64_t n)
+  {
+    return (uint)floor(log10(n) + 1);
+  }
+  string generate_stars(uint n)
+  {
+    uint i = 0;
+    string stars = "";
+    for (i = 0; i < n; i++)
+    {
+      stars += "*";
+    }
+    return stars;
+  }
+  string to_string(uint n)
```

Comment:
In the cvmfs sources in util/string.h we have a utility function to convert an `int` to an `std::string`